.travis.yml

```diff
@@ -12,10 +12,19 @@ python:
   - pypy
 install: travis_retry pip install tox-travis
 services:
-  - mongodb
+  #- mongodb
   - redis-server
 before_script:
+  # work-around to make travis-ci working with mongod 3.4
```

Comment:
please keep an eye on travis-ci and submit a PR when the work-around will not be needed anymore

Reply:
Yes I will, but I do not think it will happen any soon!